tests/command_line/test_cosym.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
 
-import os
+from pathlib import Path
```

Comment:
I generally prefer to import the module and then use `pathlib.Path` in the code then it is obvious where it comes from - I think this is generally how we are using `pathlib` throughout our source code.
```suggestion
import pathlib
```